examples/sai-i2s-passthru.rs

```diff
@@ -39,7 +40,8 @@ const APP: () = {
     }
 
     #[init]
-    fn init(ctx: init::Context) -> init::LateResources {
+    fn init(mut ctx: init::Context) -> init::LateResources {
+        utilities::logger::init();
```

Comment:
I guess this is what causes the panic in `device::CorePeripherals::take()` further down, but in any case it looks good now!